vendor/models/choice.py

```diff
@@ -42,6 +44,252 @@ class TransactionTypes(models.IntegerChoices):
     VOID = 40, _('Void')
     REFUND = 50, _('Refund')
 
+
 class TermDetailUnits(models.IntegerChoices):
     DAY = 10, _("Day")
-    MONTH = 20, _("Month")
\ No newline at end of file
+    MONTH = 20, _("Month")
+
+
```

Comment:
looks good but wondering if we can outsource country list maintenance to another django package?

Reply:
For now, I took the countries that AuthorizeNet supports. But I can look into it in the future